dienst2/templates/registration/logged_out.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base_bootstrap3.html" %}
+{% load static bootstrap3 %}
+
+{% block title %}Login{% endblock %}
+
+{% block content %}
+    <img id="logo" src="{% static 'img/logo.svg' %}" width="600" height="188" class="img-responsive center-block"/><br/>
```

Comment:
Misschien moet deze naar `/` of `accounts/login/` linken zodat je makkelijk weer terug kan inloggen?